src/main/java/core/basesyntax/CompareTwoAuto.java

```diff
@@ -1,9 +1,10 @@
 package core.basesyntax;
 
-/**
- * <p>Имплементируйте необходимый функциональный интерфейс и его метод для сравнения мощности
- * двигателей у двух авто (enginePower) и возврата разницы между ними.
- * Метод должен принимать Integer.</p>
- */
-public class CompareTwoAuto {
+import java.util.function.BinaryOperator;
+
+public class CompareTwoAuto implements BinaryOperator<Integer> {
+    @Override
+    public Integer apply(Integer power1, Integer power2) {
+        return power1 - power2;
```

Comment:
please see the doc: https://mate-academy.github.io/jv-program-fulltime/code_style_requirements.html

Reply:
done